### PR TITLE
release notes for edge-21.6.1

### PR DESCRIPTION
**Pull request by dadjeibaah**
Signed-off-by: Dennis Adjei-Baah <dennis@buoyant.io>

### CODE REVIEW

**Review by olix0r — APPROVED** [review]

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
 # Changes
 
+## edge-21.6.1
+
+This release adds support for retrying HTTP/2 requests with small (<64KB)
+message bodies, allowing the proxy to properly buffer message bodies when
+responses are classified as a failure.
```
Comment (olix0r) [inline CHANGES.md:7]: Maybe worth calling out a link to the service profile documentation.

**Review by alpeb — COMMENTED** [review]
Can you add my healthckeck fix, as it's user-facing? Something like `Expanded the "linkerd-existence" health check to also check for the destination pod readiness`

**Review by alpeb — APPROVED** [review]

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
 # Changes
 
+## edge-21.6.1
+
+This release adds support for retrying HTTP/2 requests with small (<64KB)
+message bodies, allowing the proxy to properly buffer message bodies when
+responses are classified as a failure. Documentation on how to configure
+retries can be found [here](https://linkerd.io/2.10/tasks/configuring-retries/)
```
Comment (kleimkuhler) [inline CHANGES.md:8]:
```suggestion
retries can be found [here](https://linkerd.io/2.10/tasks/configuring-retries/).
```

**Review by kleimkuhler — APPROVED** [review]
🎉

**Pull request merged**